test/flint/gfp-test.jl

```diff
@@ -1,3 +1,14 @@
+function test_elem(R::Nemo.GaloisField)
+   return rand(R, fmpz(0):characteristic(R))
```

Comment:
`rand(R)` works here too :)

Reply:
Weird, I could have sworn I tried this and had issues. Anyway, of course that's much better, thanks!